nrespipe/utils.py

```diff
@@ -55,8 +57,8 @@ def get_md5(filepath):
     return md5
 
 
-def need_to_process(filename, checksum, db_address):
-    record = dbs.get_processing_state(filename, checksum, db_address)
+def already_processed(filename, checksum, db_address):
+    record = dbs.get_processing_state(filename, db_address)
     return not record.processed or checksum != record.checksum
```

Comment:
I'm slightly confused by this logic. An image is considered "already processed" if its db record's processed field is False or the incoming checksum isn't equal to the db's checksum? `need_to_process` seems like a better name for this function. Or am I just utterly confused?